src/analyze/analyze.c

```diff
@@ -1092,7 +1099,7 @@ static int analyze_time(int argc, char *argv[], void *userdata) {
         return 0;
 }
 
-static int graph_one_property(sd_bus *bus, const UnitInfo *u, const char* prop, const char *color, char* patterns[], char* from_patterns[], char* to_patterns[]) {
+static int graph_one_property(sd_bus *bus, const UnitInfo *u, const char* prop, const char *color, const char *style, char* patterns[], char* from_patterns[], char* to_patterns[]) {
```

Comment:
i think it's time to break this long long line... i.e. one line per parameter

Reply:
line broken